extension/github-notifications-preview.js

```diff
@@ -105,6 +107,35 @@ async function openDropdown({currentTarget: indicator}) {
 		empty(container);
 		container.append(...list);
 
+		delegate('.NPG-dropdown button', 'click', async event => {
```

Comment:
This is required in the next version of `delegate-it` coming in #99 

```suggestion
		delegate(document, '.NPG-dropdown button', 'click', async event => {
```